core/backend/src/BackendHubAccess.ts

```diff
@@ -188,18 +188,27 @@ export interface CreateNewIModelProps extends IModelNameArg {
  * Generally direct access to these methods should not be required, since higher-level apis are provided.
  * @note This interface is implemented in another repository. Any changes made to this interface must be validated against
  * the implementation found here: https://github.com/iTwin/imodels-clients/blob/main/itwin-platform-access/imodels-access-backend/src/BackendIModelsAccess.ts
```

Comment:
```suggestion
```

Remove the note. The idea that we could freely break this API at any time was misguided, because the implementation lives outside of repo defining the interface and follows different versioning cadence (and we can't control what versions of each get combined in a given app).